src/components/api-reference/scoring.js

```diff
@@ -1,28 +1,1004 @@
   render() {
+    // This is necessary to disable the default styles
+    Tabs.setUseDefaultStyles(false);
+
     return (
       <div id='scoring-engine'>
         <h1 className='ds-h1'>Scoring</h1>
-        <p className='ds-text--lead'>Scores are calculated by a scoring engine package. This functionality is not yet publicly exposed.</p>
-        <p><em>Disclaimer:</em> Scoring is subject to change, based on periodic policy updates, eligibility reviews, and technical integration developments.</p>
-        <h2 className='ds-h2'>Method Signature</h2>
-        <p className='ds-text--lead'>Improvement activities (IA), advancing care information (ACI), and quality measures are scored differently. The scoring engine package used provides one scoring engine that scores and combines these three categories.</p>
-        <div className='markup markup--html'>
-          <pre className='ds-u-border--1 ds-u-padding--1'>
-            {`score(submission, provider, benchmarks)
-`}
-          </pre>
+        <ul>
+          <li><a href={`#${SCORE_RESOURCE.id}`}>{SCORE_RESOURCE.title}</a></li>
+          <li><a href={`#${SCORE_PART_RESOURCE.id}`}>{SCORE_PART_RESOURCE.title}</a></li>
+          <li><a href={`#${CATEGORY_SCORE_RESOURCE.id}`}>{CATEGORY_SCORE_RESOURCE.title}</a></li>
+          <li><a href={`#${MEASUREMENT_SET_SCORE_PART_RESOURCE.id}`}>{MEASUREMENT_SET_SCORE_PART_RESOURCE.title}</a></li>
+          <li><a href={`#${MEASUREMENT_SCORE_PART_RESOURCE.id}`}>{MEASUREMENT_SCORE_PART_RESOURCE.title}</a></li>
+          <li><a href={`#${MEASUREMENT_SCORE_RESOURCE.id}`}>{MEASUREMENT_SCORE_RESOURCE.title}</a></li>
+        </ul>
+        <h1 className='ds-h1'>Overview</h1>
+        <p className='ds-text--lead'>
+          The Scoring Engine resides within the QPP Submissions API application and calculates a performance score when it receives QPP submission data. By sending a submission through the Submissions API, the scoring engine evaluates the contained: provider profile information, measurement set performance data, and available benchmarking data, for each performance category.
+        </p>
+        <p className='ds-text--lead'>
+          A performance score is generated in two different ways. Firstly, submission by GET request with the identifier of a stored submission to the Submissions API’s submissions endpoint located at <code>/submissions/:id/score</code> will produce a score. Secondly, submission by POST request with a full submission in QPP JSON format to the Submissions API’s score preview endpoint located at <code>/submissions/score-preview</code> will also produce a score.
+        </p>
+        <p className='ds-text--lead'>
+          Next, each performance category is individually processed and scored by evaluating the corresponding measurement sets. Processing “metadata" and "messages" attached to the scored measurement sets and measurements are also compiled to generate a Score Object.
```

Comment:
Looks like you may be using the left quote here rather than the plain quote

Reply:
Thanks!